### ISSUE TITLE
Sort By Set

### ISSUE THREAD

**Issue by KakitaShiro** [Alsciende/fiveringsdb#19]
Please allow users to sort cards by set. Useful for those of us who file cards that way.

**Comment by KakitaShiro** [comment]
Any movement on this?

**Comment by Alsciende** [comment]
Not really. Cards are uncorrelated from packs now. So I would sort cycles, then packs in each cycle, then cards by position in each pack. Cards would be duplicated when they appear in 2 packs (like promos). Basically it would be a whole new page, not a sort option in the list of cards.

**Comment by KakitaShiro** [comment]
I don't want to filter cards, that is easy enough, I want to see and option for deck lists and the deckbuilder to show cards in order of card release.

**Comment by Alsciende** [comment]
That's not really a different matter. I can't sort a list of cards by pack. I'll probably implement something later on. Right now I don't know how to solve this easily.
